AY2021S1-CS2103-T16-3/tp
# Make toggle-split switch to the students tab

### What this does
This PR changes the behaviour of `toggle-split` so that it always switches to the students tab, which is probably what most users would expect (see #144). This behaviour is also now documented in the UG to avoid any confusion.

### How to test
While the students and rooms tabs are separate:
Run `toggle-split` from the students tab, rooms tab and bin tab. Verify that the combined tab is always shown after running the command.

While the students and rooms tabs are combined:
Run `toggle-split` from the combined tab and the bin tab. Verify that the students tab is always shown after running the command.
